sheet_to_triples/rdf.py

```diff
@@ -108,9 +108,13 @@ def graph_from_model(model):
     return g
 
 
+def _maybe_bnode(s, p, o):
+    return type(s) == rdflib.term.BNode or type(o) == rdflib.term.BNode
+
+
 def update_model_terms(model, triples):
     model['terms'].extend(
-        dict(subj=str(s), pred=str(p), obj=str(o)) for s, p, o in triples)
+        dict(subj=str(s), pred=str(p), obj=str(o)) for s, p, o in triples if not _maybe_bnode(s, p, o))
```

Comment:
Why filter here, rather than in `normalise_model()` which handles most of the rest of the filtering? Do we also want it on a non-normalised operation? If so, might be better to make a public filter function and use directly.

Reply:
The rationale is, why are BNode strings a useful thing to have in the model at all? It's just a reference to a phantom class that RDF would composite down into a sensible definition when outputting the model in e.g. .ttl, but our string conversion means we lose the useful part of that information -- we can't convert back into BNodes once they're in the model as strings. 

Also, how would you filter a BNode out of the model when it's just an alphanumeric string at that point? `normalise_model` doesn't do anything with the graph, which is where the "is this a BNode?" information is stored.

Reply:
Okay, fair enough, it is easier to do this before the string cast, indeed.